src/CommunityToolkit.Maui.MediaElement/MediaElement.shared.cs

```diff
@@ -377,6 +389,24 @@ public void Stop()
 		Handler?.Invoke(nameof(StopRequested));
 	}
 
+	/// <summary>
+	/// Restores screen position.
+	/// </summary>
+	public void RestoreScreen()
```

Comment:
Is this method only used to revert back from Full Screen?

If so, let's be a bit more specific with the method name. Maybe something like `RevertFromFullScreen`?

Reply:
Method was renamed as suggested from ```RestoreScreen()``` to ```RevertFromFullScreen()```